yetanotherco/aligned_layer
# refactor: grafana dashboard

# Improve grafana layout and bug fixes on metrics querying

## Description

- Show dashboard with a better distribution

- Some queries were using counters incorrectly

## How to Test

1. Run all the components

2. `make run_metrics`

3. Go to grafana dashboard `localhost:3000`

4. Grafana should show metrics correctly

## Type of change


## Checklist
